src/crimson/os/seastore/cache.h

```diff
@@ -166,6 +166,28 @@ class Cache {
     }
   }
 
+  /**
+   * get_extent_if_cached
+   *
+   * Returns extent at offset if in cache
+   */
+  Transaction::get_extent_ret get_extent_if_cached(
```

Comment:
i still think it'd be easier to understand if this method could return a tuple or a pair of `get_extent_ret` and `CachedExtentRef`.

Reply:
Using an out param and return code simplifies the logic around between Cache::get_extent and Transaction::get_extent and I want to leave it this way.